README.md

```diff
@@ -10,43 +10,44 @@ The following steps will install the `tigerbeetle-node` module to your current w
 
 > Your operating system should be Linux (kernel >= v5.6) or macOS. Windows support is not yet available but is in the works.
 
-### YARN Package Manager
-
+### YARN Package Manager Installation
 ```shell
-# Run the following from this directory:
-yarn postinstall #Download NodeJS Headers, Install Zig, Build TigerBeetle
-yarn && yarn build && yarn add
+yarn add tigerbeetle-node
 ```
 or
+### NPM Package Manager Installation
+```shell
+npm install tigerbeetle-node
+```
+
+**Development**
 
-### NPM Package Manager
-Run the following command from this directory:
+Follow these steps to get up and running when cloning the repo:
 ```shell
-npm install .
+git clone --recurse-submodules https://github.com/coilhq/tigerbeetle-node.git
+yarn
 ```
 
-**Yarn - Run Test**
-Ensure TigerBeetle (`init` & `start`) is running on the port configured in `test.ts`, then run: 
+***Yarn - Run Test***
+```shell
+# Run the following from this directory:
+yarn postinstall #Download NodeJS Headers, Install Zig, Build TigerBeetle
+yarn && yarn build
```

Comment:
@DonChangfoot Hmm, perhaps we should change the label. As we need zig and node C headers for the `yarn build` to work. 
Otherwise you get;
```
command not found: zig/zig
```

Reply:
Fixed.